dashboard/db/migrate/20170403190323_add_using_text_mode_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsingTextModeToUsers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :users, :using_text_mode, :boolean
```

Comment:
Hmm, I'm hesitant to add this to the `users` table... As I envision the need for more `user_preferences`, I'm wondering if we should be creating such a new table.